rinja_derive/Cargo.toml

```diff
@@ -33,4 +33,10 @@ once_map = "0.4.18"
 proc-macro2 = "1"
 quote = "1"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-syn = "2"
+syn = "2.0.3"
+
+[dev-dependencies]
+console = "0.15.8"
+similar = "2.6.0"
+prettyplease = "0.2.20"
+syn = { version = "2.0.3", features = ["extra-traits", "full"] }
```

Comment:
Feel free to tell me if you think this is overkill. I thought "lets make it pretty", but maybe I overdid it a bit :)

Reply:
It's dev dependencies, so I definitely don't care. :)